Duckling/Time/EN/Rules.hs

```diff
@@ -1118,6 +1118,30 @@ ruleIntervalFromDDDDMonth = Rule
       _ -> Nothing
   }
 
+ruleIntervalFromDDDDOfMonth :: Rule
```

Comment:
Added this rule because it was failing too early when trying to add "of" in the above rule.

Reply:
This is the right way to go. `ruleIntersectOf` might get in the way when disambiguating but adding more corpus examples should help. Make sure to re-generate the classifiers with the latest code and include them in the commit.